Plugin.php

```diff
@@ -330,6 +336,58 @@ protected function extendWinterPagesPlugin(): void
         }
     }
 
+    /**
+     * Extend the Winter.Sitemap plugin
+     */
+    protected function extendWinterSitemapPlugin(): void
+    {
+        $pluginManager = PluginManager::instance();
+        if (!$pluginManager->exists('Winter.Sitemap')) {
+            return;
+        }
+
+        $typeMapping = [
+            MLPage::class => ['cms-page'],
+        ];
+
+        if ($pluginManager->exists('Winter.Pages')) {
+            $typeMapping[MLPage::class] = array_merge($typeMapping[MLPage::class], ['static-page', 'all-static-pages']);
+        }
+
+        if ($pluginManager->exists('Winter.Blog')) {
+            $typeMapping[MLBlogCategoryModel::class] = ['blog-category', 'all-blog-categories'];
+            $typeMapping[MLBlogPostModel::class] = ['blog-post', 'category-blog-posts', 'all-blog-posts'];
+        }
+
+        Event::listen('winter.sitemap.processMenuItems', function ($item, $url, $theme, $apiResult) use ($typeMapping) {
+            foreach ($typeMapping as $class => $types) {
+                if (in_array($item->type, $types)) {
+                    return $class::resolveMenuItem($item, $url, $theme);
+                }
+            }
+
+            return false;
+        });
+
+        Event::listen('winter.sitemap.makeUrlSet', function ($definition, $xml, $urlSet) {
+            $urlSet->setAttribute('xmlns:xhtml', 'http://www.w3.org/1999/xhtml');
```

Comment:
Why is this required? Could this not just be the default?

Reply:
Because it's only required when we have xhtml:link nodes (which only the multilang requires for now).

Reply:
Is there any reason to not include it when we don't? Any downside to it?

Reply:
Not that I know of, other than principle.

Reply:
Let's just set it as the default in the sitemap plugin then

Reply:
done